src/galaxy.cpp

```diff
@@ -722,6 +733,179 @@ void SelectPlayerView::clickPlayer(int x, int y, int arg) {
 	exitView();
 }
 
+PlanetsListView::PlanetsListView(const GameState *game) : _game(game) {
+	Widget *w = NULL;
+	const uint8_t *pal;
+
+	_bg = gameAssets->getImage(PLANET_ARCHIVE, ASSET_PLANETLIST_BG);
+	_planetAssets = gameAssets->getImage(COLONY_VIEW_PLANET_LIST_ARCHIVE, COLONY_VIEW_PLANET_LIST_PALETTE);
```

Comment:
According to the `#define` above, `_bg` and `_planetAssets` are the same asset (`plntsum.lbx:0`). Why create two references in the same class instance?